src/header.asm

```diff
@@ -1,16 +1,26 @@
 ;
-; iNES header
+; NES2.0 header
+; https://www.nesdev.org/wiki/NES_2.0
 ;
 
-; This iNES header is from Brad Smith (rainwarrior)
+; iNES header adapted from Brad Smith (rainwarrior)
 ; https://github.com/bbbradsmith/NES-ca65-example
 
 .segment "HEADER"
 
 .include "constants.asm" ; for INES_HEADER
 
 INES_MIRROR = 0 ; 0 = horizontal mirroring, 1 = vertical mirroring (ignored in MMC1)
-INES_SRAM   = 1 ; 1 = battery backed SRAM at $6000-7FFF
+INES_SRAM = SAVE_HIGHSCORES ; 1 = battery backed SRAM at $6000-7FFF
```

Comment:
we should probably add a `HAS_SRAM` flag for clarity

Reply:
Should that flag be a new addition, or should it replace an existing flag? I think replacing `SAVE_HIGHSCORES` would make the most sense here. The header should ideally be a set-and-forget kind of thing in my opinion.

Reply:
I think it should be an additional flag so the saving highscores can be disabled separately if needed.

In a newer version we might decide to remove or replace it and leaving the name makes it easier to remove.

Reply:
I added the `HAS_SRAM` flag but disabling it is currently lumped with the `-s` build flag.